### PR TITLE
Effie ban patch 1

### PR DESCRIPTION
**Pull request by EffieBan**

### CODE REVIEW

**Comment by EffieBan** [comment]
This is a mistake pull. Sorry.

File: blog/Arrival on Goteborg.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Arrival on Goteborg 
+
+___Full of passion and hope, we landed on Goteborg, Sweden. It almost took us 15 hours arriving here. There are many flights you can choose to take. The direct flight is the one from Beijing to Stockholm, but then you need to transfer from there to Goteborg by train. For us, we brought 7 luggages so that we took the flight from Shanghai to Goteborg with direct transfer of luggages, stop in Helsinki for 2 hours. The price of the flight is around 7000 RMB.___
```
Comment (chuanjin) [inline blog/Arrival on Goteborg.md:3]: 7000RMB per person?

**Review by chuanjin — CHANGES_REQUESTED** [review]

**Comment by chuanjin** [comment]
Please put all photoes under **images** folder, and maybe you want to add mosaic for your kid.

**Comment by EffieBan** [comment]
ok，wow, I think I need to learn it later. All things new to me. Haha

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by chuanjin** [comment]
- It is better to keep only one pull request and close the other one since they are the same.
- And remove the Effiesimages and keep the one inside **images** folder so that we only keep one copy.

**Pull request closed without merging**